cosmoz-image-viewer.js

```diff
@@ -232,13 +232,13 @@
 		},
 
 		_createImageOverlay() {
-			let dialog = document.createElement('cosmoz-image-viewer-overlay');
+			const dialog = document.createElement('cosmoz-image-viewer-overlay');
 			dialog.id = 'cosmoz-image-viewer-overlay';
 			dialog.noCancelOnOutsideClick = true;
 			dialog.loop = this.loop;
 			dialog.showDetach = this.showDetach;
 			document.body.appendChild(dialog);
-			imageOverlay = dialog;
+			imageOverlay = this.imageOverlay = dialog;
```

Comment:
Didn't you removed `this.imageOverlay` in another PR ?
There is no `imageOverlay` property in this element

Reply:
@plequang I did but it is needed for tests and I omitted that aspect at that moment.

Reply:
@plequang I have added:
get imageOverlay() {
       return imageOverlay;
},